pages/shows/index.vue

```diff
@@ -1,18 +1,17 @@
 <template>
   <div>
-    <div class="m-4 text-center title">
-      <h1>Lahmacun Shows</h1>
-    </div>
-    <!-- Make a listing component -->
     <AutoCompleteSearch
       v-model='arcsiShows'
       suggestionAttribute='name'
     />
-    <ShowsLister :shows='arcsiShowsList' />
-    <div class="m-4 text-center title">
-      <h1>Past Shows</h1>
+    <div v-if='arcsiShowsList  > 0' >
```

Comment:
Nice, good to have the check, not to break stuff if server is down or similar. Maybe also to add some message with v-else?